project/glossary.md

```diff
@@ -0,0 +1,46 @@
+<!-- vim: ft=markdown colorcolumn=72
+-->
+# Glossary
+
+This glossary is a dictionary of terms defined as they are used by the
+spire/helm-charts project.  Writing the definitions of terms here helps
+to keep other documents concise and precise.  Documenting terminology
+here helps prevent misundersandings, facilitating easy onboarding of new
+team members.
+
+**Deployment**
+: A use of the Helm Charts to describe a single SPIRE cluster.
+
+**Deployment Type**
+: One of three supported deployments of the template: Standalone, 
+Primary, or Secondary.
+
+**Federation**
+: When one cluster is configured to trust elements of another cluster
+containing a different trust domain.
+
+**Federated Deployment**
+: A Primary Deployment or Standalone Deployment that is configured
+to federate with one or more Deployments.
+
+**Primary Deployment**
+: A deployment of the Helm Charts where the cluster's purpose is to
+provide certificates to Secondary Deployments.
+
+**Secondary Deployment**
+: A deployment of the Helm Charts where the cluster obtains an
+intermediate CA from a Primary Deployment and uses it to service
+Workload Identity requests.
+
```

Comment:
Honest question. I could go either way. Primary/Secondary totally works. Our charts are a different way of deploying spire, so different terminology may be the bast way to go to disambiguate things. But spire also has its own terminology that may be better to reuse. They would call it root and nested.
https://spiffe.io/docs/latest/architecture/nested/readme/

Whats the best approach there?